### ISSUE TITLE
Not working with RN 0.62

### ISSUE THREAD

**Issue by StarryFire** [mloureiro/react-native-expandable-text#21]
```
Error: Unable to resolve module `AccessibilityInfo` from `node_modules/@mloureiro/react-native-expandable-text/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: AccessibilityInfo could not be found within the project.
```
